### DISCUSSION TITLE
Virtual keypad feature addon

### DISCUSSION THREAD

**Discussion by evolv30101** [Dilbert66/esphome-dsckeybus#148] [Ideas]
Was thinking as i was verifying user slots / codes on an existing system it sure would be sweet if the user slots showed the username (from yaml) as well as the slot number similar to how the zones do.  It sure would make it easy for someone would has a lot of names in the list.  Obviously this wont be very beneficial most residential systems with 1-4 users. 

Again, appreciate all your help getting me running!

**Comment by Dilbert66** [comment]
Good point. I should have done that. I'll add it. I'm always open to suggestions!

**Comment by Dilbert66** [comment]
Ok, i've pushed an update that cleans up the log display a bit and includes the user code with the name display

**Comment by Dilbert66** [comment]
Ok, yes, I get what you wanted.  The user code program *5 section now will show the user name from the yaml as well as the code id.  That should simplify your task of editing /reviewing large lists.   Code is now pushed to dev.

**Comment by evolv30101** [comment]
Awesome.  I'll push it up and let you know after i finish tweaking my notifications.  1 thing at a time :)

But it sounds like you understood what i meant.

**Comment by Dilbert66** [comment]
I just applied a small fix so if you pushed it already you'll need to do it again.
